src/main/webapp/css/one-page-wonder.css

```diff
@@ -0,0 +1,160 @@
+/*!
```

Comment:
do we need both one-page-wonder.css and one-page-wonder.min.css? maybe just include the minified version?

Reply:
good point, I deleted the longer version